.github/workflows/validate_urls.yml

```diff
@@ -0,0 +1,39 @@
+name: Validate URLs
+
+on:
+  page_build:
+  schedule:
+    - cron: '00 14,23 * * *'
```

Comment:
Suggestion: add a comment explaining the cron syntax. It's not intuitive and I wouldn't expect the majority of maintainers to get it right away

Reply:
Added comments.  Not that everyone uses it, but the GitHub website editor gives you a hover comment to explain when it runs:
![Screen Shot 2021-11-08 at 5 16 51 PM](https://user-images.githubusercontent.com/79471462/140838679-48f0b80b-de89-4a7a-9127-b57e250316a8.png)

Reply:
That's really cool! The comment is still useful, since I know our maintainers mostly use VS code.